yarpl/include/yarpl/flowable/Subscriber.h

```diff
@@ -21,40 +21,6 @@ class Subscriber : public virtual Refcounted, public yarpl::enable_get_ref {
   virtual void onNext(T) = 0;
 };
 
-// codemod all things that inherit from Subscriber to inherit from
```

Comment:
Is it possible to make the *impl(..) methods protected?
No one should call them but the classes can override them.

Reply:
Furthermore, why don't we just provide empty implementation for them?

Reply:
No default implementation is provided for common methods, because explicit > implicit. The compiler won't let you forget to implement a pure virtual method.